recipe/meta.yaml

```
@@ -39,7 +39,6 @@ test:
 about:
   home: https://github.com/alexmojaki/eval_type_backport/
   license: MIT
-  license_url: https://github.com/alexmojaki/eval_type_backport/blob/main/LICENSE.txt
   license_family: MIT
   summary: Like `typing._eval_type`, but lets older Python versions use newer typing features.
```

make the linter happy. I think the backticks cause the error of `no summary` 
```suggestion
  summary: Like typing._eval_type, but lets older Python versions use newer typing features.
```